Core/Http/IHttpRequest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AeroGear.Mobile.Core.Http
+{
+    public static class Constants
+    {
+        const String CONTENT_TYPE_HEADER = "Content-Type";
+        const String JSON_MIME_TYPE = "application/json";
+    }
+
+    /// <summary>
+    /// Interface for requests to HTTP services.
+    /// </summary>
+    public interface IHttpRequest
```

Comment:
Any objections of using I prefix for interfaces? It's quite common in .NET world. I actually like to use personally in Java too and especially in Kotlin.

Reply:
I think it is fine for c# as it is a convention that is used by the standard c# libs. We don't use it in java because it is not a convention that has adopted by JDK.

Reply:
Ok, that's settled.